### PR TITLE
Align all the plugins with a similar TOC

### PR DESCRIPTION
**Pull request by rrick-vmw**
Add a few more details/descriptions

### CODE REVIEW

File: network-service/README.md
Comment (rezhil-vmw) [inline network-service/README.md:1]: LGTM

**Review by rezhil-vmw — APPROVED** [review]

File: vm-image-service/README.md [hunk 1]
```diff
@@ -33,71 +33,30 @@ This would be a summary of what this tutorial covers, example deploy a vm.....
...
-- Managed by : indicates whether the VM is part of a Tanzu Kubernetes Cluster
-- Address : IP address
-- Age : How long the VM has existed
-
-![VM List](source/images/vm-service-list.png "VM List")
-
-
-## Day 2 operations
+- Virtual Machine Images
+    - List all Virtual Machine Images
```
Comment (sandesh-vmw) [inline vm-image-service/README.md:42]: "...associated with the accessed namespace"?

File: vm-image-service/README.md [hunk 2]
```diff
@@ -33,71 +33,30 @@ This would be a summary of what this tutorial covers, example deploy a vm.....
...
+        - Show details of image
```
Comment (sandesh-vmw) [inline vm-image-service/README.md:43]: "...Health status, and recent events"

File: vm-image-service/README.md
Comment (sandesh-vmw) [inline vm-image-service/README.md:1]: Do we include a section to point out the fact that "there are crud operations which CCI does not support/not available for developer role"?
Things like: create/delete/modify library, Library-NS association/disassociation, etc.

**Review by sandesh-vmw — APPROVED** [review]
I've left a few comments which can be optionally addressed.

File: vm-image-service/README.md (hunk 1 above)
Comment (rrick-vmw) [reply vm-image-service/README.md:42]: We can add more clarifying statements in each of the sections. We would want to think about where the right place is to provide that overall guidance (it's all ns based).

File: vm-image-service/README.md (hunk 2 above)
Comment (rrick-vmw) [reply vm-image-service/README.md:43]: Will look at this in the next iteration.

**Pull request merged**